components/dashboard/src/user-settings/Integrations.tsx

```diff
@@ -504,13 +504,8 @@ export function GitIntegrationModal(
         onAuthorize?: (payload?: string) => void;
     },
 ) {
-    const callbackUrl = (host: string) => {
-        // Negative Lookahead (?!\/)
-        // `\/` matches the character `/`
-        // "https://foobar:80".replace(/:(?!\/)/, "_")
-        // => 'https://foobar_80'
-        host = host.replace(/:(?!\/)/, "_");
-        const pathname = `/auth/${host}/callback`;
+    const callbackUrl = () => {
+        const pathname = `/auth/callback`;
```

Comment:
Would it make sense to push this into GitpodHostUrl?

Reply:
Let's do that.